### PR TITLE
Attributes: fix tabIndex on <img> in IE11

### PR DESCRIPTION
**Pull request by Queeniebee**
Fixes gh-2647

### CODE REVIEW

File: src/attributes/prop.js [hunk 1]
```diff
@@ -55,10 +56,19 @@ jQuery.extend( {
 	propHooks: {
 		tabIndex: {
 			get: function( elem ) {
-				return elem.hasAttribute( "tabindex" ) ||
-					rfocusable.test( elem.nodeName ) || elem.href ?
-						elem.tabIndex :
-						-1;
+
+			    // elem.tabIndex doesn't always return the
```
Comment (dmethvin) [inline src/attributes/prop.js]: Do we need a `// Support` comment?

File: src/attributes/prop.js [hunk 2]
```diff
@@ -55,10 +56,19 @@ jQuery.extend( {
...
+			    // correct value when it hasn't been explicitly set
+			    // http://fluidproject.org/blog/2008/01/09/getting-setting-and-removing-tabindex-values-with-javascript/
```
Comment (dmethvin) [inline src/attributes/prop.js]: This link seems broken for me?

Comment (mzgol) [inline src/attributes/prop.js]: I'll switch it to the Web Archive one in a followup. :) We have the same link on compat so we might need to fix both.

File: src/attributes/prop.js (hunk 1 above)
Comment (mzgol) [inline src/attributes/prop.js]: Good remark, I'll add it later.

Comment (mgol) [inline src/attributes/prop.js]: Fixed in 9cb89bf91d034ec7166d9215e2f80fa765292975.

File: src/attributes/prop.js (hunk 2 above)
Comment (mgol) [inline src/attributes/prop.js]: Fixed in 9cb89bf91d034ec7166d9215e2f80fa765292975.

File: src/attributes/prop.js (hunk 1 above)
Comment (markelog) [inline src/attributes/prop.js]: I wonder how long your todo list, if "later" is four months :)

Comment (mgol) [inline src/attributes/prop.js]: Too long. :-) I'm working on that!

**Pull request closed without merging**